### DISCUSSION TITLE
Thesaurus

### DISCUSSION THREAD

**Discussion by han-solo318** [MiragianCycle/OVIWrite#13] [Q&A] [answered]
In the README, the following is mentioned:

> Flawless Composition: Spellcheck, thesaurus, and dictionary tools integrated for a seamless writing experience

but I'm not sure how to access any thesaurus or dictionary there. Which plugin(s) allow this?

**Comment by sbishop333** [comment, marked as answer]
Hi,
There is a table of available plugins on the web page, but you are
right:  not all of the ones you want are listed there ... and it is not
up-to-date with the latest iteration of OVIWrite on GitHub. Because
things change, you should open Lazy and inspect the list of available
plugins, both loaded and not loaded. On my system there are 3 plugins
that appear to be what you are looking for and they are all configured
(I think) to be manually loaded:  cmp-dictionary, LanguageTool.nvim,
and thesaurus_query.vim.

Pick one and go to ~/.config/nvim/lua/plugins and open the plugin file.
For example, here are the contents of the thesaurusquery.lua file:

return {
{ "ron89/thesaurus_query.vim" },
}

That's it. This code just tells Lazy to install the plugin; If you want
this plugin to load automatically at startup, add "lazy = false" to the
file like so:

return {
{ "ron89/thesaurus_query.vim", lazy = false },
}

You will probably have to close OVIWrite and restart it for the change
to take effect. Do that and then open Lazy again to confirm that
thesaurus_query.vim is loaded.

Now what? Now you have to learn how to use the features of the plugin;
that means going to the Github site for each and seeing what the plugin
does and how you can control it. Following up on thesaurus_query.vim,
if you open ron89/thesaurus_query.vim you will find everything (and
more) that you need to know about how to use this plugin. You will have
to go through the same process for each plugin you want to master.

I'm still working on telescope and nvim-treesitter!

Bonne chance!

SB

**Reply by han-solo318** [reply]
My issue was that I had downloaded the 0.4 release rather than from the `main` branch, so I didn't see those files in my plugins. Sorry for all the messages, and thank you for being so helpful.
